src/objects/LOD.js

```diff
@@ -77,15 +77,16 @@ class LOD extends Object3D {
 	removeLevel( distance ) {
 
 		const levels = this.levels;
-		levels.map( ( _level, index ) => {
+		for ( let i = 0; i < levels.length; i ++ ) {
```

Comment:
Can we do a reverse loop? In this case, it is no necessary to modify the index inside the loop.

Besides, would it be possible to early out when the level has been found? What is the use case for having more than on LOD for a  single distance value?

Reply:
```js
for ( let i = levels.length - 1; i >= 0; -- i ) {
```

Reply:
But like I said before if there is no compelling use case for more than one LOD per level, the loop direction does not matter. You would return after the removal and that's it.

Reply:
I think the method should look like so:
```js
removeLevel( distance ) {

	const levels = this.levels;

	for ( let i = 0; i < levels.length; i ++ ) {

		if ( levels[ i ].distance === distance ) {

			const removedElements = levels.splice( i, 1 );
			this.remove( removedElements[ 0 ].object );

			return true;

		}

	}

	return false;

}
```
Would that work for you?

Reply:
The change is done and tested